app/layout.tsx

```diff
@@ -2,10 +2,12 @@ import type { Metadata } from "next";
 import "./globals.css";
 import Header from "@/components/Header";
 import Providers from "@/providers/Providers";
+import Head from "next/head";
 
 export const metadata: Metadata = {
   title: "Myco Chat",
-  description: "Generated by create next app",
+  description: "Intelligent assistants for your digital art. Zora compatible. Styled with tailwindcss. Copy and paste. Open Source. Based.",
```

Comment:
Is this `duplicated` across multiple files?
- if yes please follow the principle of DRY.